config/PropertyNames.xml

```diff
@@ -195,363 +195,364 @@ generated from the section label.
+        <property name="Override.Secure" languages="all" />
+        <property name="Package.[any]" languages="java" />
+        <property name="Plugin.[any]" languages="cpp,csharp,java" />
+        <property name="PluginLoadOrder" languages="cpp,csharp,java" />
+        <property name="PreferIPv6Address" languages="cpp,csharp,java" default="0" />
+        <property name="PreloadAssemblies" languages="csharp" default="0" />
+        <property name="PrintAdapterReady" languages="cpp,csharp,java" />
+        <property name="PrintProcessId" languages="cpp,csharp" />
+        <property name="PrintStackTraces" languages="cpp" default="0" />
+        <property name="ProgramName" languages="cpp,csharp,java" />
+        <property name="RetryIntervals" languages="all" default="0" />
+        <property name="ServerIdleTime" languages="cpp,csharp,java" default="0" />
+        <property name="SOCKSProxyHost" languages="cpp,csharp,java" />
+        <property name="SOCKSProxyPort" languages="cpp,csharp,java" default="1080" />
+        <property name="StdErr" languages="cpp,csharp,java" />
+        <property name="StdOut" languages="cpp,csharp,java" />
+        <property name="SyslogFacility" languages="cpp,java" default="LOG_USER" />
+        <property name="ThreadPool.Client" class="threadpool" languages="cpp,csharp,java" />
+        <property name="ThreadPool.Server" class="threadpool" languages="cpp,csharp,java" />
+        <property name="ThreadPriority" languages="cpp,csharp,java" />
+        <property name="ToStringMode" languages="all" default="Unicode" />
+        <property name="Trace.Admin.Properties" languages="cpp,csharp,java" default="0" />
+        <property name="Trace.Admin.Logger" languages="cpp,csharp,java" default="0" />
+        <property name="Trace.Locator" languages="all" default="0" />
+        <property name="Trace.Network" languages="all" default="0" />
+        <property name="Trace.Protocol" languages="all" default="0" />
+        <property name="Trace.Retry" languages="all" default="0" />
+        <property name="Trace.Slicing" languages="all" default="0" />
+        <property name="Trace.ThreadPool" languages="cpp,csharp,java" default="0" />
+        <property name="UDP.RcvSize" languages="cpp,csharp,java" />
+        <property name="UDP.SndSize" languages="cpp,csharp,java" />
+        <property name="TCP.Backlog" languages="cpp,csharp,java" />
+        <property name="TCP.RcvSize" languages="cpp,csharp,java" />
+        <property name="TCP.SndSize" languages="cpp,csharp,java" />
+        <property name="UseOSLog" languages="cpp" default="0" />
+        <property name="UseSyslog" languages="cpp,java" default="0" />
+        <property name="UseSystemdJournal" languages="cpp" default="0" />
+        <property name="Warn.AMICallback" languages="cpp,csharp,java" default="1" />
+        <property name="Warn.Connections" languages="all" default="0" />
+        <property name="Warn.Datagrams" languages="all" default="0" />
```

Comment:
Warn.Datagrams in JS even though we don't support UDP?

Reply:
PR bug